posthog/queries/test/test_retention.py

```diff
@@ -1028,13 +1030,13 @@ def test_timezones(self):
             self.assertEqual(
                 pluck(result_pacific, "values", "count"),
                 [
-                    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
-                    [1, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+                    [1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+                    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
                     [0, 0, 0, 0, 0, 0, 0, 0, 0],
                     [0, 0, 0, 0, 0, 0, 0, 0],
                     [0, 0, 0, 0, 0, 0, 0],
-                    [0, 0, 0, 0, 0, 0],
-                    [1, 1, 0, 0, 0],  # person 2 is across two dates in US/Pacific
+                    [1, 1, 0, 0, 0, 0],
+                    [1, 0, 0, 0, 0],  # person 2 is across two dates in US/Pacific
```

Comment:
move comment up as well?

Q: Why the test change?

Reply:
Original test was incorrect I believe. If the person is showing up in the second interval they should be also in the first interval in the subsequent cohort